src/main/java/nextstep/qna/domain/Metadata.java

```diff
@@ -0,0 +1,62 @@
+package nextstep.qna.domain;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+public class Metadata {
+
+    private final LocalDateTime createdDate;
+    private LocalDateTime updatedDate;
```

Comment:
소소한 의견인데, 변수명도 통일성 있게 가면 좋을 것 같습니다 🙇 
```suggestion
    private LocalDateTime createAt;
    private LocalDateTime updatedAt;
```